### PR TITLE
Added Dockerfile and changed README for Dockerfile

### PR DESCRIPTION
**Pull request by AlekKras**
I decided to create a Docker version of that for people who might want to include multiple traps within a virtual private cloud as an example. I also added a section in `README.md` with simple instructions.
@Plazmaz

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:8
+
+#copy files to the /var/www/html directory
+COPY . /var/www/html/
```
Comment (Plazmaz) [inline Dockerfile]: I would prefer using another path that is not a standard web root (such as `/opt/mongodb-honeyproxy`)

File: README.md [hunk 2]
```diff
@@ -16,3 +18,24 @@ cd MongoDB-HoneyProxy
...
+
+1) Build a docker image:
+
+```
+docker build --tag="changeme" .
+```
+2) Run a docker image:
+
+```
+docker run -d -p 80:80 --name="changeme" changeme
```
Comment (Plazmaz) [inline README.md]: This project runs on port 27017 by default. Nothing is running on port 80.

**Review by Plazmaz — CHANGES_REQUESTED** [review]
This is a good idea, but I have a few comments.

File: Dockerfile (hunk 1 above)
Comment (AlekKras) [reply Dockerfile]: Just changed it :) @Plazmaz

**Review by Plazmaz — APPROVED** [review]
Looks good, thanks!

**Pull request merged**